### PR TITLE
Add final underscore to the entries

### PR DESCRIPTION
**Pull request by MathieuDutSik**
This addresses issue #719

### CODE REVIEW

File: examples/crowd-funding/src/contract.rs [hunk 1]
```diff
@@ -389,7 +389,7 @@ impl CrowdFunding {
     }
 
     // TODO(#719): rename into `initialization_argument()` after `#[derive(GraphQLView)]` is fixed.
```
Comment (ma2bd) [inline examples/crowd-funding/src/contract.rs:391]: You can remove the corresponding TODO!

**Review by ma2bd — APPROVED** [review]
Thanks!

File: linera-views-derive/src/util.rs [hunk 2]
```diff
@@ -15,6 +15,12 @@ pub fn string_to_ident(s: &str) -> Ident {
     syn::Ident::new(s, Span::call_site())
 }
 
+pub fn get_graphql_pair_for_underscore(id: &Ident) -> (Ident, String) {
```
Comment (ma2bd) [inline linera-views-derive/src/util.rs:18]: get_graphql_identifiers ?

File: examples/crowd-funding/src/contract.rs (hunk 1 above)
Comment (MathieuDutSik) [reply examples/crowd-funding/src/contract.rs:391]: Ok.

**Pull request merged**